content/cloud/custom-domains-tutorial.md

```diff
@@ -0,0 +1,73 @@
+title = "Apply Custom Domain"
+date = "2023-07-26T12:00:00Z"
+template = "cloud_main"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/cloud/custom-domains-tutorial.md"
+
+---
+
+- [Prerequisites](#prerequisites)
+- [Select Your Spin Application](#select-your-spin-application)
+- [Input Your Custom Domain](#input-your-custom-domain)
+- [Update Your Domain Registrar With Fermyon Cloud Name Servers](#update-your-domain-registrar-with-fermyon-cloud-name-servers)
+  - [Repeat Delegation Only: Update Domain Registrar with TXT Record](#repeat-delegation-only-update-domain-registrar-with-txt-record)
+- [Verify Domain Has Been Delegated](#verify-domain-has-been-delegated)
+- [Next Steps](#next-steps)
+
+Custom domains enable you to bring your own domain to Fermyon Cloud.  Unlike generic or randomly generated domains, a custom domain reflects the brand, identity, or purpose of the workload. For a conceptual overview of custom domains on Fermyon Cloud, visit (Custom Domains on Fermyon Cloud)[./custom-domain.md]. This tutorial will walk you through how to apply a custom domain to your Spin application running on Fermyon Cloud. At the end of this tutorial, you will have delegated your domain to Fermyon Cloud. Let's get started!
+
+## Prerequisites
+
+To ensure the tutorial goes smoothly, please ensure you have the following: 
+
+* Most importantly, you must have a domain you own. If you don't have a domain, please visit a domain registrar like [Namecheap](https://www.namecheap.com/) or [Google Domains](https://domains.google) to purchase one. 
+* A Fermyon Cloud account that is set up via your preferred [GitHub user account](https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/remembering-your-github-username-or-email)
+* A Spin application you would like to apply the custom domain to. If you aren't familiar with how to deploy an application on Fermyon Cloud, check out our [develop a Spin application](./develop.md) article to learn how to build a Spin application and then our [deploy a Spin application article](./deploy.md) for deployment instructions. 
```

Comment:
Maybe clarify that this is a _deployed_ application.  "Spin application" could be local.

```suggestion
* A Spin application already deployed to Fermyon Cloud that you would like to apply the custom domain to. If you aren't familiar with how to deploy an application on Fermyon Cloud, check out our [develop a Spin application](./develop.md) article to learn how to build a Spin application and then our [deploy a Spin application article](./deploy.md) for deployment instructions. 
```